WordPress/src/main/java/org/wordpress/android/ui/prefs/SummaryEditTextPreference.java

```diff
@@ -0,0 +1,63 @@
+package org.wordpress.android.ui.prefs;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.preference.EditTextPreference;
+import android.support.annotation.NonNull;
+import android.text.TextUtils;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+
+/**
+ * Standard EditTextPreference that has attributes to limit summary length.
```

Comment:
I'm confused about the purpose of this class. Perhaps the comment could explain it better?